pkg/controller/servicebindingrequest/servicebinder_test.go

```diff
@@ -573,3 +607,68 @@ func TestServiceBinder_Bind(t *testing.T) {
 		},
 	}))
 }
+
+func TestInitApplicationSelector(t *testing.T) {
```

Comment:
the method under test is called now `ensureDefaults` so the test function name could be updated as well.

Reply:
Done. a1da3ae1bb026f31d5ed2a5ff3e173f01628024f